src/main/java/core/basesyntax/dao/impl/SmileDaoImpl.java

```diff
@@ -12,16 +15,42 @@ public SmileDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Smile create(Smile entity) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(entity);
```

Comment:
```suggestion
            session.save(entity);
```
https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_02.html

Reply:
done